@here/harp-datasource-protocol/lib/InterpolatedProperty.ts

```diff
@@ -119,7 +130,7 @@ export function getPropertyValue<T>(
     return getInterpolatedLength(property, level, pixelToMeters);
 }
 
-function getInterpolatedLength(
+export function getInterpolatedLength(
```

Comment:
Why do you need to export ``getInterpolatedLength`` and ``getInterpolatedColor``? ``getPropertyValue`` is hiding these details from the user, which might not know how to properly interpolate the value of a certain property.

Reply:
Accident.
Removed exports in next patch.